var/spack/repos/builtin/packages/esmf/package.py

```diff
@@ -263,7 +264,13 @@ def edit(self, spec, prefix):
 
             # PIO-dependent features will be enabled and will use the
             # PIO library that is included and built with ESMF.
-            os.environ['ESMF_PIO'] = 'internal'
+            if spec.satisfies('@:8.3.0'):
```

Comment:
Is that a up to 8.3.0 but not including?

Reply:
This might be incorrect